tests/kubernetes/graph/test_local_graph.py

```diff
@@ -0,0 +1,30 @@
+import os
+
+from checkov.kubernetes.graph_builder.local_graph import KubernetesLocalGraph
+from checkov.kubernetes.parser.parser import parse
+from tests.kubernetes.graph.base_graph_tests import TestGraph
+
+TEST_DIRNAME = os.path.dirname(os.path.realpath(__file__))
+
+
+class TestKubernetesLocalGraph(TestGraph):
```

Comment:
Can/Should we add more tests for failing k8s files?

Reply:
I think we should add tests for failing k8s files for the runner (when implemented), this part isn't effected by this because we are not running any checks here.